packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -571,12 +567,7 @@ export class FluidDataStoreRuntime
 	 * 2. Attaching the graph if the data store becomes attached.
 	 */
 	public bindToContext() {
-		if (this.bindState !== BindState.NotBound) {
-			return;
-		}
-		this.bindState = BindState.Binding;
-		this.dataStoreContext.bindToContext();
-		this.bindState = BindState.Bound;
+		this.makeVisibleAndAttachGraph();
```

Comment:
If we call `this.dataStoreContext.makeLocallyVisible()` directly, multiple calls to this would result in multiple calls to data store context. Calling makeVisibleAndAttachGraph makes sure that it updates the visibility state and data store context won't be called multiple times.

With this change, storing its handle (basically call to attachGraph()) and call to bindToContext() are equivalent. This was the case before as well, just that the state machine was divergent.